modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc

```diff
@@ -1,36 +1,75 @@
 // Module included in the following assemblies:
 //
-// * monitoring/cluster_monitoring/managing-cluster-alerts.adoc
+// * monitoring/managing-alerts.adoc
 
 [id="getting-information-about-alerts-and-alerting-rules_{context}"]
-= Getting information about alerts and alerting rules
+= Getting information about alerts, silences and alerting rules
 
-You can find an alert and see information about it or its governing alerting rule.
+The Alerting UI provides detailed information about alerts and their governing alerting rules and silences.
+
+.Prerequisites
+
+* You have access to the cluster as a developer or as a user with view permissions for the project that you are viewing metrics for.
 
 .Procedure
 
+*To obtain information about alerts in the Administrator perspective*:
+
 . Open the {product-title} web console and navigate to the *Monitoring* -> *Alerting* -> *Alerts* page.
 
-. Optional: Filter the alerts by name using the *Filter Alerts by name* field.
+. Optional: Search for alerts by name using the *Name* field in the search list.
 
-. Optional: Filter the alerts by state using one or more of the state buttons *Firing*, *Silenced*, *Pending*.
+. Optional: Filter alerts by state, severity, and source by selecting filters in the *Filter* list.
 
-. Optional: Sort the alerts by clicking one or more of the *Name*, *Severity*, and *State* column headers.
+. Optional: Sort the alerts by clicking one or more of the *Name*, *Severity*, *State*, and *Source* column headers.
 
-. After you see the alert you need, you can see either details of the alert or details of its governing alerting rule.
-+
-To see alert details, click on the name of the alert. The page that opens has the graph with timeseries of the alert. It also has information about the alert, including:
+. Select the name of an alert to navigate to its *Alert Details* page. The page includes a graph that illustrates alert time series data. It also provides information about the alert, including:
 +
 --
-* A link to its governing alerting rule
-* Description of the alert
-* Labels attached to the alert
+** A description of the alert
+** Messages associated with the alerts
+** Labels attached to the alert
+** A link to its governing alerting rule
+** Silences for the alert, if any exist
 --
+
+*To obtain information about silences in the Administrator perspective*:
+
+. Navigate to the *Monitoring* -> *Alerting* -> *Silences* page.
+
+. Optional: Filter silences by state by selecting filters in the *Filter* list.
+
+. Optional: Sort the silences by clicking one or more of the *Name*, *Firing Alerts*, and *State* column headers.
+
+. Select the name of a silence to navigate to its *Silence Details* page. The page includes a description of the silence, scheduling information, and a list of related firing alerts.
```

Comment:
This duplicates text from the "Getting information about silences" section below. More specifically, this text is a less comprehensive version of the "Getting information about silences" section.

Could we remove this to avoid the duplication? Or maybe it's better to remove the "Getting information about silences" section and merge its  additional info in here?

Reply:
Thank you for highlighting this. I have now removed the 'Getting information about silences' section and merged the details into this one, as per your suggestion.

Reply:
👍 Thank you!